blocks/identity-block/features/login/default.jsx

```diff
@@ -0,0 +1,161 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from '@arc-fusion/prop-types';
+import { isServerSide } from '@wpmedia/engine-theme-sdk';
+import { useFusionContext } from 'fusion:context';
+import getProperties from 'fusion:properties';
+import getTranslatedPhrases from 'fusion:intl';
+import ReCaptcha from 'react-google-recaptcha';
+
+import FormInputField, { FIELD_TYPES } from '../../components/FormInputField';
+import HeadlinedSubmitForm from '../../components/HeadlinedSubmitForm';
+import useIdentity from '../../components/Identity';
+
+import './styles.scss';
+
+const Login = ({ customFields, arcSite }) => {
+  let { redirectURL } = customFields;
+  const {
+    redirectToPreviousPage,
+    loggedInPageLocation,
+    forgotPasswordURL,
+    signupURL,
+  } = customFields;
+
+  const { isAdmin } = useFusionContext();
+  const { locale = 'en' } = getProperties(arcSite);
+  const phrases = getTranslatedPhrases(locale);
+
+  const { Identity, isInitialized } = useIdentity();
+
+  const [recaptchaToken, setRecaptchaToken] = useState();
+  const [recaptchaConfig, setRecaptchaConfig] = useState({
+    signinRecaptcha: false,
+    status: 'initial',
+  });
```

Comment:
can we move the recaptcha work to another pr? Looks like we're hitting test coverage limits and removing this would help and the scope
```suggestion
```

Reply:
yep, sounds good. I'll pull it out

Reply:
I thought re-captcha was deferred for MVP as well.